src/OpenTelemetryShopwareBundle.php

```diff
@@ -24,6 +24,8 @@
  */
 class OpenTelemetryShopwareBundle extends AbstractBundle
 {
+    protected string $extensionAlias = 'open_telemetry_shopware';
```

Comment:
@shyim @keulinho few questions to you.

1. Should we keep default naming, or change it before releasing v.2.0.0? (this will influence key name in configuration)

2. Should we use this as default namespace for metrics? `->defaultValue('io.opentelemetry.contrib.php.shopware')`.

Reply:
is there a need to change it? 🤔

Reply:
It's rather a last call before releasing v0.2.0 Maybe we would prefer something like `shopware_open_telemetry`, or have established extension naming approach I am not aware of. With this release we introduce bundle related configuration, changing extension alias afterwards would be more problematic.

Regarding default metrics namespace I do not see a reason to change it, but still can do it before release.